.test-infra/jenkins/job_PerformanceTests_Python.groovy

```diff
@@ -18,46 +18,105 @@
-      output: 'gs://temp-storage-for-end-to-end-tests/py-it-cloud/output'
-  ]
-  def pipelineArgList = []
-  pipelineArgs.each({
-    key, value -> pipelineArgList.add("--$key=$value")
-  })
-  def pipelineArgsJoined = pipelineArgList.join(',')
-
-  def argMap = [
-      beam_sdk                 : 'python',
-      benchmarks               : 'beam_integration_benchmark',
-      bigquery_table           : 'beam_performance.wordcount_py_pkb_results',
-      beam_it_class            : 'apache_beam.examples.wordcount_it_test:WordCountIT.test_wordcount_it',
-      beam_it_module           : 'sdks/python',
-      beam_prebuilt            : 'true',  // skip beam prebuild
-      beam_python_sdk_location : 'build/apache-beam.tar.gz',
-      beam_runner              : 'TestDataflowRunner',
-      beam_it_timeout          : '1200',
-      beam_it_args             : pipelineArgsJoined,
-  ]
-
-  commonJobProperties.buildPerformanceTest(delegate, argMap)
+
+class PerformanceTestConfigurations {
+  String jobName
+  String jobDescription
+  String jobTriggerPhrase
+  String buildSchedule = 'H */6 * * *'  // every 6 hours
+  String benchmarkName = 'beam_integration_benchmark'
+  String sdk = 'python'
+  String bigqueryTable
+  String itClass
+  String itModule
+  Boolean skipPrebuild = false
+  String pythonSdkLocation
+  String runner = 'TestDataflowRunner'
+  Integer itTimeout = 1200
+  Map extraPipelineArgs
+}
+
+
+def testConfigurations = [
+    new PerformanceTestConfigurations(
+        jobName           : 'beam_PerformanceTests_Python',
+        jobDescription    : 'Python SDK Performance Test',
+        jobTriggerPhrase  : 'Run Python Performance Test',
+        bigqueryTable     : 'beam_performance.wordcount_py_pkb_results',
+        skipPrebuild      : true,
+        pythonSdkLocation : 'build/apache-beam.tar.gz',
+        itClass           : 'apache_beam.examples.wordcount_it_test:WordCountIT.test_wordcount_it',
+        itModule          : 'sdks/python',
+        extraPipelineArgs : [
+            project         : 'apache-beam-testing',
+            staging_location: 'gs://temp-storage-for-end-to-end-tests/staging-it',
+            temp_location   : 'gs://temp-storage-for-end-to-end-tests/temp-it',
+            output          : 'gs://temp-storage-for-end-to-end-tests/py-it-cloud/output',
+        ],
+    ),
+    new PerformanceTestConfigurations(
+        jobName           : 'beam_PerformanceTests_Python35',
+        jobDescription    : 'Python35 SDK Performance Test',
+        jobTriggerPhrase  : 'Run Python35 Performance Test',
+        bigqueryTable     : 'beam_performance.wordcount_py35_pkb_results',
+        skipPrebuild      : true,
+        pythonSdkLocation : 'test-suites/dataflow/py35/build/apache-beam.tar.gz',
+        itClass           : 'apache_beam.examples.wordcount_it_test:WordCountIT.test_wordcount_it',
+        itModule          : 'sdks/python/test-suites/dataflow/py35',
+        extraPipelineArgs : [
```

Comment:
The extraPipelineArgs seems identical in both jobs. Do we consider pulling them out as a defalutPipelineArgs?

Reply:
They are same since both of them benchmark `wordcountIT` with `TestDataflowRunner`. We probably can reuse some of them that's required by `TestDataflowRunner` which could be more commonly used among other benchmarks.

Reply:
grouped them into `dataflowPipelineArgs` in new commit. PTAL